### ISSUE TITLE
Add typography styles to style.dart

### ISSUE THREAD

**Issue by marcedwards** [Western-Health-Covid19-Collaboration/wh_covid19_app#21]
This should be all or most of the text styles we need in the app. They’re all named similarly to HTML elements (H1, H2, P etc). I’m prefixing everything with text. Please let me know if that’s a good idea or not.

Please note that these styles reference the colour constants we have (listed in #9).

![textstyles](https://user-images.githubusercontent.com/1121547/77273222-21f31980-6d07-11ea-8c86-d3030b774e02.png)

```
const TextStyle textH1 = TextStyle(
  color: blackAlpha900,
  fontWeight: FontWeight.w600,
  fontStyle: FontStyle.bold,
  fontSize: 28.0,
  fontFamily: 'InterBold'
);

const TextStyle textH2 = TextStyle(
  color: blackAlpha900,
  fontWeight: FontWeight.w600,
  fontStyle: FontStyle.regular,
  fontSize: 28.0,
  fontFamily: 'Inter'
);

const TextStyle textH3 = TextStyle(
  color: blackAlpha900,
  fontWeight: FontWeight.w600,
  fontStyle: FontStyle.bold,
  fontSize: 22.0,
  fontFamily: 'InterBold'
);

const TextStyle textH4 = TextStyle(
  color: blackAlpha900,
  fontWeight: FontWeight.w600,
  fontStyle: FontStyle.regular,
  fontSize: 22.0,
  fontFamily: 'Inter'
);

const TextStyle textH5 = TextStyle(
  color: blackAlpha900,
  fontWeight: FontWeight.w600,
  fontStyle: FontStyle.semibold,
  fontSize: 17.0,
  fontFamily: 'InterSemiBold'
);

const TextStyle textP = TextStyle(
  color: blackAlpha600,
  fontWeight: FontWeight.w600,
  fontStyle: FontStyle.regular,
  fontSize: 17.0,
  fontFamily: 'Inter'
);

const TextStyle textFooter = TextStyle(
  color: blackAlpha600,
  fontWeight: FontWeight.w600,
  fontStyle: FontStyle.regular,
  fontSize: 13.0,
  fontFamily: 'Inter'
);

const TextStyle textCaption = TextStyle(
  color: blackAlpha600,
  fontWeight: FontWeight.w600,
  fontStyle: FontStyle.semibold,
  fontSize: 13.0,
  fontFamily: 'InterSemiBold'
);
```

### Questions and notes

- The project currently only contains `Inter.ttf`. We’ll need medium, semibold and bold added to the project. `Inter-Bold-slnt=0.ttf`, `Inter-Medium-slnt=0.ttf`, `Inter-SemiBold-slnt=0.ttf`, `Inter-Regular-slnt=0.ttf`.
- I’m not sure what to do with `fontWeight: FontWeight.w600`. Please advise.
- I’ve guess the values for `fontFamily` and `fontStyle`. Please advise on those as well.
- Should these be prefixed differently?

Inter is free and open source, and available here: https://fonts.google.com/specimen/Inter

**Comment by rockgecko-development** [comment]
Hi Marc
1. See https://api.flutter.dev/flutter/painting/TextStyle-class.html , scroll to "Custom Fonts" to see how to add these ttf files to the project and reference them
2. FontWeight is "The thickness of the glyphs used to draw the text", ref: https://api.flutter.dev/flutter/dart-ui/FontWeight-class.html 
thin is `w100`,
"bold" is w700.
If you'd rather just "normal" you can specify `FontWeight.normal` (which equals `w400`)
3. `FontStyle` is either `normal` or `italic` https://api.flutter.dev/flutter/dart-ui/FontStyle-class.html
4. LGTM

**Comment by domesticmouse** [comment]
Is this a PR in flight somewhere? As the `Inter` font is on our fonts.google.com CDN, I'd suggest loading the font via [google_fonts](https://pub.dev/packages/google_fonts) pub package. Thoughts?

**Comment by rockgecko-development** [comment]
@domesticmouse I caught up with Luke, I'll do this shortly. Good idea re google_fonts

**Comment by rockgecko-development** [comment]
Actually, looks like @adamkoch merged this already in #25 , so it's already done

**Comment by adamkoch** [comment]
I was going to do it, but while I added the colors I didn't add the other text styles as per above. I may try and do it as part of #30 which has a lot of text styles (but if someone else wants to do it separately please feel free to).

**Comment by adamkoch** [comment]
Sorry just saw the note to use google fonts package, I actually just included the ttf fonts directly. Please review PR:
https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/pull/55

**Comment by adamkoch** [comment]
Fixed in #55
